API/Controllers/AccountController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using VL.Shared.Interfaces;
+using VL.Shared.Model;
+
+namespace API.Controllers
+{
+    [Route("api/account")]
+    [ApiController]
+    public class AccountController : ControllerBase
+    {
+        private readonly ILogger<AccountController> _logger;
+        private readonly IAccountService _accountService;
+
+        public AccountController(
+            ILogger<AccountController> logger,
+            IAccountService accountService)
+        {
+            _logger = logger;
+            _accountService = accountService;
+        }
+
+        public static IdentityUser user = new IdentityUser();
+
+        [HttpPost]
+        [Route("register")]
+        [ProducesResponseType(typeof(bool), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesDefaultResponseType]
+        public async Task<IActionResult> Register([FromBody] UserDto userDto)
+        {
+            _logger.LogInformation($"Registration Attempt for {userDto.Email}");
+
+            var result = await _accountService.Register(userDto);
+
+            if (!result.Succeeded)
+            {
+                return BadRequest();
+            }
+
+            return Accepted();
```

Comment:
Now a user registered, you _could_ return a JWT and log them into the app and save an unnecessary step

Reply:
![image](https://user-images.githubusercontent.com/99216796/195872762-934f9a65-7e2d-4fa4-ba75-a0f9e0f66d65.png)

Reply:
This seems like what you're asking for, login following a registration. My login task will provide a JWT.

Reply:
Never call a Controller from another Controller, implement the controller's logic inside it and use services / methods to reuse similar code